### PR TITLE
Update expected Debian sources.

### PR DESCRIPTION
**Pull request by nuclearsandwich**
This looks like it was copy-pasted from the above entry for Ubuntu and
used the Ubuntu repository components rather than the Debian ones.

The inclusion of contrib and non-free comes from https://wiki.ros.org/Installation/Debian#melodic.2FInstallation.2FDebianSources.Configure_your_Debian_repositories

There is no Debian specific article for setting up ROS sources but the
Ubuntu article applies if you update the distribution name.

### CODE REVIEW

**Review by wjwwood — APPROVED** [review]
Makes sense to me, might want a second review? If not I'll let you merge @nuclearsandwich.

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**
